src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -4181,38 +3978,35 @@ void GetVRFInputData(EnergyPlusData &state, bool &ErrorsFound)
                                                      _,
                                                      _,
                                                      _,
-                                                     state.dataHVACVarRefFlow->VRFTU(VRFTUNum).HeatingCapacitySizeRatio);
-                            } else if (state.dataHVACVarRefFlow->VRF(state.dataHVACVarRefFlow->VRFTU(VRFTUNum).VRFSysNum).HeatingCapacitySizeRatio >
-                                       1.0) {
-                                SetDXCoolingCoilData(
-                                    state,
-                                    state.dataHVACVarRefFlow->VRFTU(VRFTUNum).HeatCoilIndex,
-                                    ErrorsFound,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    _,
-                                    state.dataHVACVarRefFlow->VRF(state.dataHVACVarRefFlow->VRFTU(VRFTUNum).VRFSysNum).HeatingCapacitySizeRatio);
+                                                     thisVrfTU.HeatingCapacitySizeRatio);
+                            } else if (state.dataHVACVarRefFlow->VRF(thisVrfTU.VRFSysNum).HeatingCapacitySizeRatio > 1.0) {
+                                SetDXCoolingCoilData(state,
```

Comment:
There are many crazy things in EnergyPlus, but this is top 5. I want to set a single field, so I need to set up and call a function with 20 parameters. Yes, that makes sense.